inc/search.class.php

```diff
@@ -7589,6 +7589,10 @@ static function makeTextSearchValue($val) {
             ($matches[1] != '^' ? '%' : '') .
             trim($matches[2]) .
             ($matches[3] != '$' ? '%' : '');
+      } else if (isset($matches[1])
```

Comment:
Maybe add a comment on the whole code to explain different cases

Reply:
Ok, will do it

Reply:
I added a comment line 7595